velox/functions/sparksql/Register.cpp

```diff
@@ -388,6 +388,27 @@ void registerFunctions(const std::string& prefix) {
 
   VELOX_REGISTER_VECTOR_FUNCTION(udf_make_timestamp, prefix + "make_timestamp");
 
+  registerFunction<TimestampToMicrosFunction, int64_t, Timestamp>(
+      {prefix + "unix_micros"});
+  registerFunction<MicrosToTimestampFunction, Timestamp, int8_t>(
+      {prefix + "timestamp_micros"});
+  registerFunction<MicrosToTimestampFunction, Timestamp, int16_t>(
+      {prefix + "timestamp_micros"});
+  registerFunction<MicrosToTimestampFunction, Timestamp, int32_t>(
+      {prefix + "timestamp_micros"});
+  registerFunction<MicrosToTimestampFunction, Timestamp, int64_t>(
```

Comment:
could you add a helper function in `RegistrationHelpers.h` to simplify these code?

Reply:
Thanks, I added a helper function, it's only used in this class, so I put it same file.

Reply:
Maybe, better to make the added helper function generic and move to RegistrationHelpers.h?
```
template <template <class> class T,  typename TReturn>
void registerUnaryIntegral(const std::vector<std::string>& aliases) {
   registerFunction<T, TReturn, int8_t>(aliases);
   ...
}
```